bibliopixel/animation/sequence.py

```diff
@@ -47,4 +48,4 @@ def step(self, amt=1):
                 self.index = 0
 
         if not self.completed and self.animations:
-            self.current_animation.run_all_frames()
+            self.current_animation.start()
```

Comment:
Sequence animations are always threaded?!  So what if a sequence contains a sequence?  How many threads are there??

Urg, well, the whole threading thing was pretty terminal anyway, so I'm not going to hold this up, but it all got just a little bit more obscure.  :-D